apps/global_notice/views.py

```diff
@@ -0,0 +1,22 @@
+from django.utils import timezone
+from rest_framework import permissions, viewsets
+
+from .models import GlobalNotice
+from .permissions import GlobalNoticePermission, IsGlobalNoticeAthenticated
+from .serializers import GlobalNoticeSerializer
+
+
+class GlobalNoticeViewSet(viewsets.ModelViewSet):
```

Comment:
Global notice는 어드민이 어드민 페이지에서만 만들 수 있게 할 계획이어서 `ReadOnlyModelViewSet`을 사용하면 되겠습니다. (테스트 수정 필요해요.)

```suggestion
class GlobalNoticeViewSet(viewsets.ReadOnlyModelViewSet):
```

Reply:
https://github.com/sparcs-kaist/new-ara-api/pull/432/commits/c46a07b0e8dfc113001f515cdfee04ff6768a08e
Readonly로 변경하면서 필요없어진 permission, test들 수정했습니다